packages/communication-react/CHANGELOG.beta.md

```diff
@@ -4,99 +4,87 @@ This log was last generated on Tue, 27 Jun 2023 22:07:18 GMT and should not be m
 
 <!-- Start content -->
 
-## [1.7.0-beta.1](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.7.0-beta.1)
+## [1.7.0-beta.1](https://github.com/azure/communication-ui-library/tree/1.7.0-beta.1)
 
-Tue, 27 Jun 2023 22:07:18 GMT 
-[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.6.0...@azure/communication-react_v1.7.0-beta.1)
+Tue, 27 Jun 2023 22:07:18 GMT
+[Compare changes](https://github.com/azure/communication-ui-library/compare/1.6.0...1.7.0-beta.1)
 
-### Minor changes
+### Mention - Public Preview
+The Web UI SendBox and MessageThread components now support mentions capabilities in chat message thread. Enables user to @ mention another user by providing a Search capability to look for the desired user and then tagging them on the message.
 
-- `@azure/communication-react`
+### File Sharing
+The Web UI Chat composite now supports to view and download a file shared by a Teams user.
```

Comment:
@jimchou-dev  and @palatter Please provide more details about File Sharing feature if needed. Thanks.